integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -7669,6 +7669,18 @@ msgstr "Nur Koordinaten übernehmen"
 msgid "Please choose one option or cancel"
 msgstr "Bitte eine Option auswählen oder abbrechen"
 
+#: cms/templates/pois/poi_form_sidebar/related_contacts_box.html
+msgid "Related contacts"
+msgstr "Zugehörige Kontakte"
+
+#: cms/templates/pois/poi_form_sidebar/related_contacts_box.html
+msgid "This location is currently referred to in those contacts."
+msgstr "Dieser Ort wird in folgenden Kontakten verwendet."
+
+#: cms/templates/pois/poi_form_sidebar/related_contacts_box.html
+msgid "This location is not currently referred to in any contact."
+msgstr "Zur Zeit wird dieser Ort in keinem Kontakt verwendet."
```

Comment:
```suggestion
msgstr "Zur Zeit gibt es keine Kontakte zu diesem Ort."
```

Nit: I think this might read a bit easier, but not too important I'd say